packages/cactus-plugin-satp-hermes/grafana/provisioning/alerting/contact-points.yaml

```diff
@@ -0,0 +1,21 @@
+apiVersion: 1
```

Comment:
this configuration should be injectable via the root configuration file. possibly the main configuration file refers to a contact points definition (we may save the example here). Please make it clear that the current file is an example